tests/components/backup/test_manager.py

```diff
@@ -485,7 +493,7 @@ async def test_async_initiate_backup_with_agent_error(
     assert result["event"] == {
         "manager_state": BackupManagerState.CREATE_BACKUP,
         "stage": None,
-        "state": CreateBackupState.COMPLETED,
+        "state": CreateBackupState.FAILED,
```

Comment:
Note that the backup will be marked as failed now if there are agent errors.

Reply:
Yes, I think that's fine.